yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@after-work.js/aw@6.0.14":
   version "6.0.14"
-  resolved "https://registry.yarnpkg.com/@after-work.js/aw/-/aw-6.0.14.tgz#5bc86ab8189d08ab7752542a2ae3b137827b96c0"
```

Comment:
What is going on here? All deps have change registry?

Reply:
I added then removed some dependencies, seems like that really messed up the lock-file. deleting it and rerunning yarn didn't really seem to help 😐

Reply:
The seem to use the same registry now at least. But there a still a ton of changes for just adding `globby` as a dependency.

I'm guessing that is because the lock file had not been updated properly. So I think we can ignore it.